lib/workers/pr/automerge.ts

```diff
@@ -41,7 +42,7 @@ export async function checkAutoMerge(
       prAutomergeBlockReason: PrAutomergeBlockReason.Conflicted,
     };
   }
-  if (requiredStatusChecks && pr.canMerge !== true) {
+  if (!ignoreTests && pr.canMerge !== true) {
```

Comment:
`pr.canMerge` is unrelated to test status so I'm not sure if this logic is right. It might be an existing logic flaw

Reply:
If I simply remove this check, some unit tests fails